pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -56,6 +57,18 @@ func newReconciler(mgr manager.Manager) reconcile.Reconciler {
 		scheme:           mgr.GetScheme(),
 		awsClientBuilder: &awsclient.Builder{},
 	}
+
+	configMap, err := controllerutils.GetOperatorConfigMap(reconciler.client)
+	if err != nil {
+		log.Error(err, "failed retrieving configmap")
+	}
+
+	fr, ok := configMap.Data["fedramp"]
+	if !ok {
+		log.Error(err, "fedramp key not available in configmap")
+	}
+	frBool, _ := strconv.ParseBool(fr)
```

Comment:
Can we check for `nil` and add a `log.Info` message for detecting if we are in the fedramp env?

Reply:
I don't think this type can be nil, as the configmap struct is a `map[string]string`, so you'd get "".  I'm not sure if an empty string plays nice in the strconv.ParseBool though.

Reply:
Yes, strings default to "" and if you use bool parse, it defaults to false

Reply:
The point was to check for an error condition here (if any).